test/image_test.py

```diff
@@ -627,6 +627,32 @@ def test_image_dockerfile_copy(builder_version, servicer, client, tmp_path_with_
         assert files == {"/data.txt": b"hello", "/data/sub": b"world"}
 
 
+def test_image_docker_command_entrypoint(builder_version, servicer, client, tmp_path_with_content):
+    app = App()
+    data_mount = Mount.from_local_dir(tmp_path_with_content, remote_path="/")
+    app.image = Image.debian_slim().entrypoint([])
+    app.function()(dummy)
+
+    with app.run(client=client):
+        layers = get_image_layers(app.image.object_id, servicer)
+        assert "ENTRYPOINT []" in layers[0].dockerfile_commands
+        files = {f.mount_filename: f.content for f in Mount._get_files(data_mount.entries)}
```

Comment:
I might be wrong but I think you can skip these checks and the data mount itself in both tests